extensions/ConfirmEdit/ReCaptcha.php

```diff
@@ -43,23 +34,25 @@
 
 /**
  * Make sure the keys are defined.
+ *
+ * @throws Exception
  */
 function efReCaptcha() {
-	global $wgReCaptchaPublicKey, $wgReCaptchaPrivateKey;
-	global $recaptcha_public_key, $recaptcha_private_key;
-	global $wgServer;
+	$wg = F::app()->wg;
 
-	// Backwards compatibility
-	if ( $wgReCaptchaPublicKey == '' ) {
-		$wgReCaptchaPublicKey = $recaptcha_public_key;
-	}
-	if ( $wgReCaptchaPrivateKey == '' ) {
-		$wgReCaptchaPrivateKey = $recaptcha_private_key;
+	if ( $wg->ReCaptchaPublicKey == '' || $wg->ReCaptchaPrivateKey == '' ) {
+		throw new Exception( wfMessage( 'recaptcha-misconfigured' ) );
```

Comment:
Does this Message object need to have a method called on it? Like escaped() or text()?